coops-ui/src/components/AnnualReport/PaymentErrorDialog.vue

```diff
@@ -1,38 +1,54 @@
 <template>
-  <v-dialog v-model="dialog" width="45rem">
+  <v-dialog id="payment-error-dialog" v-model="dialog" width="45rem" persistent>
```

Comment:
I like the addition of `persistent` tho to be honest, i would have assumed it to be default! Good job.